### PR TITLE
ref: corrected the dockerfile and updated the contributing.md file

### LINKED ISSUE #2406: [Refactor] Refactored project startup code for improved efficiency.
### File Name

Dockerfile

### Reason for Refactoring the Code

The previous implementation of the project startup code functioned adequately but lacked adherence to best practices for maintainability and efficiency. By refactoring the code, I've replaced the initial approach with a more robust and standardized method, ensuring improved code readability and future scalability. This update aligns with recommended practices, enhancing overall project quality and developer experience.

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues).
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md).
- [X] The changes don't break the code and don't introduce new functionality.
- [X] I am willing to work on this issue (optional).

### PR DESCRIPTION
**Pull request by JoshiUtsav**
Closes #2406 

Additionally, @rupali-codes, you need to push the Linkshub Docker image to Docker Hub so that contributors can pull it.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -306,8 +313,8 @@ git commit -m "feat(auth): Implement user signup process (#789)"
 ### Levels
 
 We came up with this chart so you can gauge the issue's difficulty and pick ones that fit within your skillset:
-| Points              | Contribution                                                                |
+| Points | Contribution |
 |---------------------|-----------------------------------------------------------------------------|
-| `priority: low`:    | Addition of new links/categories or doing any small task (e.g fixing typos) |
-| `priority: medium`: | Modifying an existing feature                                               |
-| `priority: high`:   | Making completely new feature                                               |
+| `priority: low`: | Addition of new links/categories or doing any small task (e.g fixing typos) |
```
Comment (rupali-codes) [inline CONTRIBUTING.md:318]: Remove the colon

**Review by CBID2 — APPROVED** [review]
Looks great

**Comment by JoshiUtsav** [comment]
@rupali-codes @CBID2, don't approve this. I mistakenly deleted the pnpm lock file, which might cause issues in production.

**Comment by CBID2** [comment]
Thanks for the heads up @JoshiUtsav

**Comment by Anmol-Baranwal** [comment]
@JoshiUtsav 
You can simply add it.

**Pull request closed without merging**